src/k-provider/ovp/provider.js

```diff
@@ -47,6 +48,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
         }
         const redirectFromEntryId = this._getEntryRedirectFilter(mediaInfo);
         this._dataLoader.add(OVPMediaEntryLoader, {entryId, ks, redirectFromEntryId});
+        this._dataLoader.add(OVPBumperEntryLoader, {entryId: mediaInfo.bumperId, ks, redirectFromEntryId});
```

Comment:
we always add it now?

Reply:
`OVPBumperEntryLoader` is invalid when no id so it won't be added actually